draft-ietf-jsonpath-base.md

```diff
@@ -1912,15 +1912,15 @@ Queries:
 
 ## Normalized Paths
 
-A Normalized Path is a canonical representation of the location of a node in a value and
+A Normalized Path is a minimal representation of the location of a node in a value which
```

Comment:
```suggestion
A Normalized Path is a unique representation of the location of a node in a value which
```

This is definitely a place where "unique" fits.  (Not so great in the other place further up.)